modules/fundamental/src/purl/model/details/purl.rs

```diff
@@ -275,3 +395,33 @@ impl FromQueryResultMultiModel for LicenseCatcher {
             .try_model_columns(license::Entity)
     }
 }
+
+#[derive(Debug)]
+#[allow(dead_code)] //TODO sbom field is not used at the moment, but we will probably need it for graph search
```

Comment:
Does this comment still apply? It says "field".

Reply:
It doesn't. It was copied over from another query catcher. Fixed now, thanks for spotting it.